crates/aiken-lang/src/tipo/infer.rs

```diff
@@ -83,7 +88,7 @@ impl UntypedModule {
         for def in consts.into_iter().chain(not_consts) {
             let definition = infer_definition(
                 def,
-                &name,
+                &module_name,
```

Comment:
Just renamed this because it was getting confusing down the line. `name` is a bit too ambiguous.